packages/installation-tests/screencast.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const requireName = process.argv[2];
+let success = {
+  'playwright': ['chromium', 'firefox', 'webkit'],
+  'playwright-chromium': ['chromium'],
+  'playwright-firefox': ['firefox'],
+  'playwright-webkit': ['webkit'],
+}[requireName];
+if (process.argv[3] === 'none')
+  success = [];
+if (process.argv[3] === 'all')
+  success = ['chromium', 'firefox', 'webkit'];
+
+const playwright = require(requireName);
+const path = require('path');
+const fs = require('fs');
+
+(async () => {
+  for (const browserType of success) {
+    try {
+      const browser = await playwright[browserType].launch();
+      const context = await browser.newContext();
+      const page = await context.newPage();
+      const videoFile = path.join(__dirname, requireName + '--' + browserType + '-v.webm');
+      await playwright._toImpl(page)._delegate.startScreencast({outputFile: videoFile, width: 320, height: 240});
```

Comment:
Let's use public api instead?

Reply:
ah looked at the wrong screencast test  for the inspiration 🤷‍♂️